### ISSUE TITLE
Back Animation not updated to component changes (duplicate #105)

### ISSUE THREAD

**Issue by nevaehph** [fram-x/FluidTransitions#127]
Example:

![sample](https://user-images.githubusercontent.com/5110870/47899380-5406a000-deb4-11e8-82dc-3c44d8341394.gif)

The application I am developing contains 2 pages as seen from the GIF, the latter being a Details page of sorts. Both pages contain a shared Transition component that contain images that can be swapped via a button click. The transition towards the Details page is fine, but if the image is changed to another and `this.props.navigation.goBack()` is called the animation back still retains the image prior to the change.

In the example, the image is changed from Kiwis to an Apple at the Details Page, but when transitioning back, the animation still shows the Kiwis.

The image pane of both pages are set such that altering one will change the other.  

Is the forward and back transition using the same animation layer? Is there a solution for this?

The Transition component of each page are coded as such, 

```
<Transition shared="img">
    <View style={styles.cardImageContainer}>
        <Animated.Image
            style={{
                ...styles.cardImage,
                opacity: fadeAnim
            }}

            source={{ uri: this.props.link1 }}
        />
        <Animated.Image
            style={{
                ...styles.cardImage,
                opacity: fadeAnim
            }}

            source={{ uri: this.props.link2 }}
        />
    </View>
</Transition>
```

Each Transition component contains multiple Image components that layer on top of one another using `display:"absolute"`. On a method call, each image's opacity is altered through `Animated` to change the image being displayed.

**Comment by chrfalch** [comment]
Hi, @nevaehph - thanks for submitting an issue. This issue has been raised before (#105) and the problem is by design due to performance considerations. I'll take a second look and see if it is possible. For now I'm marking it as a duplicate if #105 and will continue to track it there.

**Comment by chrfalch** [comment]
Should be fixed in the latest release 0.2.73.
